src/structures/Interaction.js

```diff
@@ -73,7 +73,7 @@ class Interaction extends Base {
      * This member permissions, if exists, in the channel this interaction was sent in
      * @type {?Permissions}
      */
-     this.memberPermissions = data.member?.permissions ? new Permissions( data.member.permissions ) : null;
+    this.memberPermissions = data.member?.permissions != null ? new Permissions(data.member.permissions) : null;
```

Comment:
Once again, please install prettier and ESLint. Also I deleted the comment because I realized that permissions is sent as a string and not an integer, which this added check wouldn't even fix, so you can keep it to `data.member?.permissions`

Reply:
Prettier unfortunately stopped functioning for me a few weeks ago — resolved the lint order issue.